### PR TITLE
Add Speakers page

### PR DESCRIPTION
**Pull request by ozasadnyy**
## Mobile

<br/>

### CODE REVIEW

File: src/elements/footer-social.html [hunk 1]
```diff
@@ -149,23 +149,23 @@
         <div class="share">
           <paper-icon-button
             class="share-gplus"
-            icon="hoverboard:gplus-alt"
```
Comment (sophieH29) [inline src/elements/footer-social.html]: Why have you changed icons?

Comment (ozasadnyy) [inline src/elements/footer-social.html]: I have renamed them as another type is more common

File: src/elements/speaker-details.html [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+  <template>
+    <style is="custom-style" include="shared-styles flex flex-alignment positioning"></style>
+
+    <style>
+
+      :host {
+        margin: 0;
+        display: block;
+        min-height: 100%;
+        min-width: 100%;
```
Comment (sophieH29) [inline src/elements/speaker-details.html]: wouldn't work just height / width?

File: src/elements/speaker-details.html [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+        border-bottom: 1px solid var(--border-light-color);
+        color: var(--secondary-text-color);
+        height: auto;
+      }
+
+      .photo {
+        margin-right: 16px;
+        width: 72px;
+        height: 72px;
+        background-color: #e8e8e8;
```
Comment (sophieH29) [inline src/elements/speaker-details.html]: might be better to move color to shared styles

File: src/elements/speaker-details.html [hunk 4]
```diff
@@ -0,0 +1,224 @@
...
+        padding: 0;
+        color: var(--secondary-text-color);
+      }
+
+      .content-list-details {
+        margin-bottom: 16px;
+      }
+
+      .content-list-icon {
+        margin: 2px 43px 0 19px;
```
Comment (sophieH29) [inline src/elements/speaker-details.html]: "magic numbers"? someone told me about 8px-grid :)

File: src/elements/speaker-details.html [hunk 5]
```diff
@@ -0,0 +1,224 @@
...
+        .header-content {
+          padding: 24px;
+        }
+
+        .content {
+          padding: 24px;
+        }
+
+        .content-list-icon {
+          margin: 2px 43px 0 27px;
```
Comment (sophieH29) [inline src/elements/speaker-details.html]: it differs with mobile only with margin-left

File: src/elements/speaker-details.html [hunk 6]
```diff
@@ -0,0 +1,224 @@
...
+          padding: 16px 0;
+        }
+
+        .header-content {
+          padding: 24px;
+        }
+
+        .content {
+          padding: 24px;
+        }
```
Comment (sophieH29) [inline src/elements/speaker-details.html]: .header-content,
.content {
padding: 24px;
}

File: src/elements/speaker-details.html [hunk 7]
```diff
@@ -0,0 +1,224 @@
...
+          viewport: {
+            type: Object,
+            statePath: 'ui.viewport'
+          }
+        }
+      }
+
+      static get observers() {
+        return [
+          '_handleClose(opened)'
```
Comment (sophieH29) [inline src/elements/speaker-details.html]: Does it observe changes to property from Redux?

File: src/hoverboard-app.html [hunk 8]
```diff
@@ -210,16 +211,16 @@ <h3 class="location">{$ location.short $}</h3>
 
         <app-header slot="header" condenses fixed effects="waterfall">
           <app-toolbar class="header" sticky>
-            <paper-icon-button icon="hoverboard:menu" aria-label="Menu" hidden$="[[!isPhoneSize]]"
+            <paper-icon-button icon="hoverboard:menu" aria-label="Menu" hidden$="[[viewport.isTabletPlus]]"
```
Comment (sophieH29) [inline src/hoverboard-app.html]: Will be the menu icon it visible on desktop?

File: src/pages/speakers-page.html [hunk 9]
```diff
@@ -1,24 +1,237 @@
...
+        <a class="speaker" href$="/speakers/[[speaker.id]]/">
+          <div relative>
+            <plastic-image
+              class="photo"
+              srcset="[[speaker.photoUrl]]"
+              sizing="cover"
+              lazy-load preload fade
+            ></plastic-image>
+            <div class="badges" layout horizontal>
+              <template is="dom-repeat" items="[[speaker.badges]]" as="badge">
```
Comment (sophieH29) [inline src/pages/speakers-page.html]: shouldn't you check badge existence as in details?

File: src/elements/speaker-details.html [hunk 10]
```diff
@@ -0,0 +1,224 @@
...
+        .header-toolbar {
+          padding: 16px 0;
+        }
+
+        .header-content {
+          padding: 24px;
+        }
+
+        .content {
+          padding: 24px;
```
Comment (sophieH29) [inline src/elements/speaker-details.html]: .header-content,
.content {
padding: 24px;
}

File: src/elements/speaker-details.html (hunk 7 above)
Comment (ozasadnyy) [inline src/elements/speaker-details.html]: `opened` is defined in `iron-overlay-behavior.html`. We pass this value from `hoverboard-app.html` and it's connected to redux store

File: src/hoverboard-app.html (hunk 8 above)
Comment (ozasadnyy) [inline src/hoverboard-app.html]: yes, >= tablet size

File: src/pages/speakers-page.html (hunk 9 above)
Comment (ozasadnyy) [inline src/pages/speakers-page.html]: no need, it will be invisible

**Comment by ozasadnyy** [comment]
Add featured speakers block on home page + added box-shadow to cards

**Pull request merged**